nbs/models.timellm.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #74.                     llm = None,
</pre><p> </p><p>If it is needed, remove default of None. Also, it is not clear from the docstring how to use it, what do users need to pass?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done! I added GPT-2 as the default LLM if the user does not specify another model.